src/Microsoft.Framework.DependencyInjection/ServiceLookup/ServiceScope.cs

```diff
@@ -8,20 +8,37 @@ namespace Microsoft.Framework.DependencyInjection.ServiceLookup
     internal class ServiceScope : IServiceScope
     {
         private readonly ServiceProvider _scopedProvider;
+        private readonly ServiceScopeFactory _parentFactory;
 
         public ServiceScope(ServiceProvider scopedProvider)
         {
             _scopedProvider = scopedProvider;
         }
+        internal ServiceScope(ServiceProvider scopedProvider, ServiceScopeFactory parentFactory)
```

Comment:
Why internal for this? will the public constructor ever be used with these changes?